ONSdigital/eq-author-app
# Remove Undelete

### What is the context of this PR?
This is to remove the undelete functionality for the app as we will no longer support undeleting as we will now be implementing a versioning system instead. This is a replacement for a previous PR that included questionnaire deletion confirmation but that is now in a separate PR

### How to review 
- Check the app works as it should
- Check you can delete all types of things (questionnaire, section, question, answer and section introduction)
- Check that when deleting these things the toast is still shown but now there is no undo button
